cmd/internal/codegen/parse/crd.go

```diff
@@ -292,6 +297,11 @@ func (b *APIs) parseArrayValidation(t *types.Type, found sets.String, comments [
 		Type:  "array",
 		Items: &v1beta1.JSONSchemaPropsOrArray{Schema: &items},
 	}
+	if t.Name.Name == "[]byte" {
+		props.Type = "string"
```

Comment:
It will be good to include the comment mentioned in the bug here for the context. It will help the reader of the code to understand the context a bit better.